azure-boards-common/src/main/java/com/synopsys/integration/azure/boards/common/http/AzureHttpServiceFactory.java

```diff
@@ -30,6 +30,8 @@
 
 public class AzureHttpServiceFactory {
     public static final String DEFAULT_BASE_URL = "https://dev.azure.com";
+    public static final String DEFAULT_AUTHORIZATION_URL = "https://app.vssps.visualstudio.com/oauth2/authorize&response_type=Assertion";
+    public static final String DEFAULT_TOKEN_URL = "https://app.vssps.visualstudio.com/oauth2/token?client_assertion_type=urn:ietf:params:oauth:client-assertion-type:jwt-bearer&grant_type=urn:ietf:params:oauth:grant-type:jwt-bearer";
```

Comment:
Are these URLs always going to be the same?

Reply:
These are the two URLs will never change for OAuth?

Reply:
As far as I can tell, yes. The base url might be different if they aren't hosted, but it's unclear how OAuth works in that case.